### DISCUSSION TITLE
plantuml diagrams in generated PDF

### DISCUSSION THREAD

**Discussion by isylhdin** [Mogztter/asciidoctor-web-pdf#586] [Q&A] [answered]
PlantUml diagrams are not displayed in the generated PDF (Images are displayed correctly) : 

```
:imagesdir: images

plantuml::images/test.puml[format=png,alt="Test",align="center"]
```
I don't get any error during the PDF generation

**Comment by ggrossetie** [comment, marked as answer]
Did you install and require `asciidoctor-kroki` (see: https://github.com/Mogztter/asciidoctor-web-pdf#asciidoctor-extensions)? 
Could you share the command you are running?

**Reply by isylhdin** [reply]
Thank you for your answer. 
Oops I forgot to install this extension.

Here is what I've tried : 
```
npm i asciidoctor-kroki --save-dev
asciidoctor-pdf --require asciidoctor-kroki doc/index.adoc -o dist/test.pdf && del-cli doc/test.html
```

I am behind a proxy and got this error : 
```
Failed to load resource: net::ERR_PROXY_CONNECTION_FAILED at https://kroki.io/plantuml/png/....
```

**Reply by ggrossetie** [reply]
If you are behind a proxy, then you will need to run Kroki locally: https://github.com/mogztter/asciidoctor-kroki#using-your-own-kroki
